### ISSUE TITLE
[Bug Report] Navigation Drawer can be tabbed into even if it is closed

### ISSUE THREAD

**Issue by phiter** [vuetifyjs/vuetify#10821]
### Environment
**Vuetify Version:** 2.2.17
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.132
**OS:** Windows 10

### Steps to reproduce
1. Open the example
2. Focus on the Toggle button
3. Press Tab

### Expected Behavior
The focus should not move to any of the items inside the navigation drawer

### Actual Behavior
Items inside are focusable.

### Reproduction Link
<a href="https://codepen.io/phiter/pen/XWbEXoj" target="_blank">https://codepen.io/phiter/pen/XWbEXoj</a>

### Other comments
This is an accessibility bug with an "easy" fix. Adding [`visibility: hidden`](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility) will make it so the element cannot receive focus, the same applies for the elements inside (unless the user adds `visibility: visible` to the children manually, in which case those children will be focusable).

We need to detect when the Navigation Drawer is fully hidden and add that style then.
